### PR TITLE
Add Tor hidden sites

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/Jackett.Common/Definitions/noname-club.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ type: public
 encoding: windows-1251
 links:
   - https://nnmclub.to/
+  - http://nnmclub5toro7u65.onion/ # will be https when there will be brave enough CA to issue certificate
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/noname-club.yml:10]: Change these all to *.onion.ly

Also, not sure the comments are necessary.

File: src/Jackett.Common/Definitions/rutor.yml [hunk 2]
```diff
@@ -8,7 +8,8 @@ encoding: UTF-8
 links:
   - http://rutor.info/ # site does not support https ERR_CONNECTION_REFUSED
   - http://rutor.is/ # site does not support https ERR_CONNECTION_REFUSED
-  - https://rutor.nocensor.space/
+  - https://rutor.nocensor.space/issue
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/rutor.yml:11]:
```suggestion
  - https://rutor.nocensor.space/
```

File: src/Jackett.Common/Indexers/RuTracker.cs [hunk 3]
```diff
@@ -33,7 +33,8 @@ public class RuTracker : BaseWebIndexer
 
         public override string[] AlternativeSiteLinks { get; protected set; } = {
             "https://rutracker.org/",
-            "https://rutracker.net/"
+            "https://rutracker.net/",
+            "http://rutrackerripnext.onion/" // will be https when there will be brave enough CA to issue certificate
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Indexers/RuTracker.cs:37]: Not working with *.onion.ly

File: src/Jackett.Common/Definitions/noname-club.yml (hunk 1 above)
Comment (galeksandrp) [reply src/Jackett.Common/Definitions/noname-club.yml:10]: - It seemed to me, looking through source code, that any non-https link should be commented with reason why it is non-https.
- Why use `onion.ly`? The point of PR was to provide single-click transition to Tor hidden sites, not tor2web.

Confirm change?

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/noname-club.yml]: `if there is a TOR domain for this, then appending .ly to the *.onion domain is an accepted way to get access.
We use this method for several indexers already.`
https://github.com/Jackett/Jackett/issues/6081#issuecomment-810031982

Vast majority of users won't be using Tor, so this supports them. As for Tor users, they just have to click and delete 3 characters.

**Comment by garfield69** [comment]
we are not going to add these .onion alternate site links.
users expect sitelinks to work using the regular Jackett setup, or with a standard proxy/VPN.
Since there is no notice that these .onion links require a TOR proxy to work, uninformed users will complain that the link does not work and open tickets.

**Pull request closed without merging**